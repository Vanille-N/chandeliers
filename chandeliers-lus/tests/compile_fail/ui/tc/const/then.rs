//! Trying to use a temporal operator in a `const` context.
chandeliers_lus::decl! {
    const X : int = 0 -> 1;
}

fn main() {}
