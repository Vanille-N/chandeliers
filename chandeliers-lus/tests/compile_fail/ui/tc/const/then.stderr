error: The later operator (-> / fby) is not valid in const contexts
 --> tests/fail/ui/tc/const/then.rs:2:21
  |
2 |     const X : int = 0 -> 1;
  |                     ^^^^^^
  |
  = note: You must put this definition inside a node
